src/application.rs

```diff
@@ -34,7 +35,14 @@ pub fn create_app() -> Result<rocket::Rocket, Box<dyn Error>> {
         Err(err) => return Err(Box::new(err)),
     };
 
+    let database_url = match app.config().get_string("database_url") {
+        Ok(database_url) => database_url,
+        Err(err) => return Err(Box::new(err)),
+    };
+    let storage: Box<dyn Storage> = Box::new(SqlStorage::new(&database_url)?);
```

Comment:
**Q:** What is `database_url`? Is it `String` or `&str`?

Reply:
`String`. That's why I need to add a `&` when passing it to `SqlStorage::new()`